build.gradle

```diff
@@ -85,6 +85,8 @@ project.ext.externalDependency = [
     'jacksonCore': 'com.fasterxml.jackson.core:jackson-core:2.13.2',
     'jacksonDataBind': 'com.fasterxml.jackson.core:jackson-databind:2.13.2.2',
     'jacksonDataFormatYaml': 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.13.2',
+    'jacksonJDK8': 'com.fasterxml.jackson.datatype:jackson-datatype-jdk8:2.13.2',
```

Comment:
Current Jackson doesn't support Java 8 optional data types

Reply:
Do we really _need_ optionals?

Reply:
It is to mark jarFileName as optional and some other configuration parameters

Reply:
Are we able to make these nullable?

Reply:
The point of this is we should strive to avoid introducing any new dependencies. We've seen how this can lead to vulns and a lot of additional work down the line